common/src/main/java/com/wynntils/mc/event/PlayerInteractEvent.java

```diff
@@ -94,6 +82,11 @@ public void setUseItem(Event.Result triggerItem) {
             this.useItem = triggerItem;
         }
 
+        @Nonnull
```

Comment:
Also this. I thought we agreed we should not have random nullness annotations sprinkled here and there?

Reply:
(Doing the "annotate for null safety" needs to be all-in or nothing, otherwise it's quite pointless. And I don't believe the payoff is worth the effort for all-in)

Reply:
This is from javax iirc. So we did not block these